courses/free-courses-id.md

```diff
@@ -139,6 +139,9 @@
 * [Belajar Golang Basic Bahasa Indonesia](https://www.youtube.com/playlist?list=PLCZlgfAG0GXDztO-BFc9R5afhP26Dhsgm) - Sekolah Koding
 * [Golang Fundamental](https://buildwithangga.com/kelas/golang-fundamental?thumbnail=nk4neM2UyG.54&main_leads=browse) - BuildWithAngga
 * [Golang Tutorial - Bahasa Indonesia](https://www.youtube.com/playlist?list=PL-CtdCApEFH-0i9dzMzLw6FKVrFWv3QvQ) - Eko Kurniawan Khannedy, Programmer Zaman Now (YouTube)
+* [Tutorial Golang Fundamental Bahasa Indonesia](https://www.youtube.com/watch?v=xzNT4JywW0A) - Agung Setiawan
+* [Tutorial Golang Web API Bahasa Indonesia - Full Course](https://www.youtube.com/watch?v=GjI0GSvmcSU) - Agung Setiawan
+* [Tutorial Golang Website Development Bahasa Indonesia](https://www.youtube.com/watch?v=K76y2_ZQYwY) - Agung Setiawan
```

Comment:
Although all of them satisfies with long video (`+ 1h`) and has chapters integrated in timeline...

:question: Are there a author Golang playlist for this? It's always better.

Reply:
"Tutorial Golang Fundamental Bahasa Indonesia" and "Tutorial Golang Website Development Bahasa Indonesia" both has a playlist version hosted in another website (for free), but in order to access it we need to have an account. As for the other courses, I cannot find their playlist version.